walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/handlers/OpenId4VCIRequestHandler.kt

```diff
@@ -0,0 +1,66 @@
+package com.microsoft.walletlibrary.requests.handlers
+
+import com.microsoft.walletlibrary.networking.entities.openid4vci.credentialmetadata.CredentialMetadata
+import com.microsoft.walletlibrary.networking.entities.openid4vci.credentialoffer.CredentialOffer
+import com.microsoft.walletlibrary.networking.operations.FetchCredentialMetadataNetworkOperation
+import com.microsoft.walletlibrary.requests.VerifiedIdRequest
+import com.microsoft.walletlibrary.util.LibraryConfiguration
+import com.microsoft.walletlibrary.util.OpenId4VciException
+import com.microsoft.walletlibrary.util.VerifiedIdExceptions
+
+internal class OpenId4VCIRequestHandler(private val libraryConfiguration: LibraryConfiguration) :
+    RequestHandler {
```

Comment:
nit: probably android thing but should this be on a new line or line 11?

Reply:
IDE formatting again. Fixed it.